notebooks/finetuning.ipynb

```diff
@@ -0,0 +1,137 @@
+    "      # Defines fields that will be filterable\n",
+    "      \"filterable\": [\"department_name\", \"colour_group_name\", \"garment_group_name\", \"index_group_name\", \"index_name\"]\n",
+    "    }\n",
+    ")\n",
+    "index.status(watch=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Finetuned Index\n",
+    "\n",
+    "Finetuning requires feedback data. Namely, examples of queries with relevance judgements on their results. The data takes the form of `(query, object_id, label)`, where `object_id` is the ID of the search result in the Object Store, and `label` is the rating on a `{BAD, OK, GREAT}` scale. \n",
+    "\n",
+    "Here are example grades of a search for dark blue polo shirt:\n",
+    "\n",
+    "```json\n",
+    "{\"object_id\": \"570004012\", \"query\": \"dark blue polo shirt\", \"label\": \"GREAT\"}\n",
+    "{\"object_id\": \"806794003\", \"query\": \"dark blue polo shirt\", \"label\": \"GREAT\"}\n",
+    "{\"object_id\": \"751334002\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"570004069\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"570004078\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"689107004\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"806794004\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"689113002\", \"query\": \"dark blue polo shirt\", \"label\": \"GREAT\"}\n",
+    "{\"object_id\": \"570004106\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "```\n",
+    "\n",
+    "[Complete example](https://d11p8vtjlacpl4.cloudfront.net/demo-data/graded_input.jsonl)\n",
+    "\n",
+    "These training labels are scoring results for a search on `dark blue polo shirt` as either `GREAT` or `BAD`. The Funetuning process will use this feedback to create a new Index optimized on the feedback. To download the full set of training labels and build a finetuned index with it:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = requests.get(\"https://d11p8vtjlacpl4.cloudfront.net/demo-data/graded_input.jsonl\")\n",
+    "training_data = [json.loads(line) for line in response.text.splitlines()]\n",
+    "ft_index = index.finetune(training_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Executing this code will kick off the Finetuning process. You will receive an email when the finetuning process is completed.\n",
+    "\n",
+    "## Compare results\n",
+    "Once you’re Finetuning is complete, you’ll be able to find your new index on the Indexes page in the Console. You’ll see the Finetuned tag on the new index.\n",
```

Comment:
Are we able to hyperlink "Console" to https://app.objective.inc/dashboard/indexes?